pydis_site/apps/api/README.md

```diff
@@ -0,0 +1,69 @@
+# The "api" app
+
+This application takes care of most of the heavy lifting in the site, that is,
+allowing our bot to manipulate and query information stored in the site's
+database.
+
+We make heavy use of [Django REST
+Framework](https://www.django-rest-framework.org) here, which builds on top of
+Django to allow us to easily build out the
+[REST](https://en.wikipedia.org/wiki/Representational_state_transfer) API
+consumed by our bot. Working with the API app requires basic knowledge of DRF -
+the [quickstart
+guide](https://www.django-rest-framework.org/tutorial/quickstart/) is a great
+resource to get started.
+
+## Directory structure
+
+Let's look over each of the subdirectories here:
+
+- `migrations` is the standard Django migrations folder. You usually won't need
+  to edit this manually, as `python manage.py makemigrations` handles this for
+  you in case you change our models.
```

Comment:
Would it be worth mentioning here our policy of renaming migrations to be human-readable/more informative? That might be diving too deep into our policy.

Reply:
Yes!!!!!!!!!!!!

Reply:
Is this the only app for which migrations are relevant? If not, we may want to have a more general documentation page to explain this, which ties into my review comment.

Reply:
I added this in 073e09a0. Thanks for the review!